app/views/details/personal-information/immigration.html

```diff
@@ -2,30 +2,33 @@
 
 {% set primaryNavId = "details" %}
 
-{% set formaction = "/application/" + "/personal-information/immigration" %}
+{% set formaction = "/details/personal-information/immigration/answer" %}
 {% set title = "Right to work or study in the UK" %}
 
 {% block beforeContent %}
   {{ govukBackLink({
-    href: (referrer if referrer else "/application/" + "/personal-information/nationality/")
+    href: "/details/personal-information/nationality"
   }) }}
 {% endblock %}
 
 {% block content %}
-
+  <div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
   <h1 class="govuk-heading-l">{{ title }}</h1>
 
-  <p class="govuk-body">You’ll need the right to work or study in the UK for the duration of your course.</p>
-
-  <p class="govuk-body"><a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa" class="govuk-link">Check if you’ve already got the right to work or study in the UK, or whether you’ll need to apply for a visa</a>.</p>
-
+  <p class="govuk-body">You can use a visa which is due to expire before the end of your course, as long as you have the right to renew it.</p>
+  <p class="govuk-body"><a href="#" class="govuk-link">Check your immigration status and whether you’ll need to apply for a visa</a>.</p>
```

Comment:
Need to add the link here?

Reply:
Yes, that needs to be added, but the previous link has been withdrawn.. So I dont know where to get a new link from for guidance about that

Reply:
Is it this: https://www.gov.uk/check-uk-visa (linked from Laura Tennant's doc)

Reply:
was about to comment the same link! I will add that